app/app_main.c

```diff
@@ -730,6 +731,23 @@ int main(int argc, char **argv)
    CHECK_ENABLE_CAP_RV(rv);
    rv = acvp_enable_prereq_cap(ctx, ACVP_KDF135_SNMP, ACVP_PREREQ_SHA, value);
    CHECK_ENABLE_CAP_RV(rv);
+
+   rv = acvp_enable_kdf135_ssh_cap(ctx, &app_kdf135_ssh_handler);
+   CHECK_ENABLE_CAP_RV(rv);
+   rv = acvp_enable_prereq_cap(ctx, ACVP_KDF135_SSH, ACVP_PREREQ_SHA, value);
```

Comment:
Make sense however, was not in the spec. in section 2.1. I will update the code.